<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QingJia</class>
 <widget class="QWidget" name="QingJia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_qj">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>851</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>请假申请</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>姓  名(*)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>部    门：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>54</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>填表日期：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_qjStartTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_qjStart">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> 职    务：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_position">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请假时间：</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_qjEnd">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_qjEndTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>到</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>210</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>请假事由(*)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_qjReason">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dealTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget_qjStartTime">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>248</width>
      <height>197</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_department">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_qjType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>如果换休，请备注换休日期</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>共请(*)：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_qjDayNum">
     <property name="geometry">
      <rect>
       <x>392</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>如：0.5天</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sj">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>伤假</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_bj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>病假</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_shijia">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>事假</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_hx">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>换休</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_other">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>其他</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_dw">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget_qjEndTime">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>248</width>
      <height>197</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>请假类型(*)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_qjRecord">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>如果换休,请备注换休日期</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>备    注：</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>lineEdit_name</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_qjStartTime</zorder>
   <zorder>toolButton_qjStart</zorder>
   <zorder>label_5</zorder>
   <zorder>comboBox_position</zorder>
   <zorder>label_6</zorder>
   <zorder>timeEdit</zorder>
   <zorder>toolButton_qjEnd</zorder>
   <zorder>timeEdit_2</zorder>
   <zorder>lineEdit_qjEndTime</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>textEdit_qjReason</zorder>
   <zorder>lineEdit_dealTime</zorder>
   <zorder>comboBox_department</zorder>
   <zorder>groupBox_qjType</zorder>
   <zorder>calendarWidget_qjEndTime</zorder>
   <zorder>label_12</zorder>
   <zorder>textEdit_qjRecord</zorder>
   <zorder>label_9</zorder>
   <zorder>calendarWidget_qjStartTime</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_leader1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>651</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>所在部门相关意见</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dealResult1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>意见(选填)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_leaderAdvice1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_leader2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>651</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>总经办意见</string>
   </property>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dealResult2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>意见(选填)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_leaderAdvice2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_leader3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>560</y>
     <width>651</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>总经理意见</string>
   </property>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>处理结果：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dealResult3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>意见(选填)：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_leaderAdvice3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pButtonOk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>660</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pButtonOk
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton#pButtonOk:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton#pButtonOk:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonCancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>660</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pButtonCancel
{
    color:black;
    background-color:rgb(238 , 238 , 238);
    border: 1px solid rgb(183 , 183 , 183);
}

QPushButton#pButtonCancel:hover
{
    color:black;
    background-color:rgb(228 , 240 , 250);
    border: 1px solid rgb(15 , 150 , 255);
}

QPushButton#pButtonCancel:pressed
{
    color:black;
    background-color:rgb(204 , 228 , 247);
    border: 1px solid rgb(1 , 84 , 153);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="closeButton" name="toolButton_close">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>0</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_allLeader">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>191</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>审核人</string>
   </property>
   <widget class="QComboBox" name="comboBox_leader1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>部门领导：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_leader2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>总经办：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>总经理：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_leader3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请假申请</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>closeButton</class>
   <extends>QToolButton</extends>
   <header location="global">closebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
